schema/config_schema_v3.9.json

```diff
@@ -242,8 +288,11 @@
             }
           ]
         },
+        "oom_kill_disable": {"type": "boolean"},
+        "oom_score_adj": {"type": "integer", "minimum": -1000, "maximum": 1000},
         "pid": {"type": ["string", "null"]},
-
+        "pids_limit": {"type": ["number", "string"]},
```

Comment:
Can we put this under `resources`?

Reply:
We are discussing re-introducing support for V2 schema attributes, this doesn't prevent another discussion about deprecating some of them and introduce a better way to define equivalent feature